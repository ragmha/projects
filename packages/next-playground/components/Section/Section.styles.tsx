import styled from 'styled-components'

export const StyledSection = styled.section`
  padding-bottom: 20px;
`
