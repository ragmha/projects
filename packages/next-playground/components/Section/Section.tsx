import { FC } from 'react'

import { StyledSection } from './Section.styles'

export const Section: FC = ({ children }) => (
  <StyledSection>{children}</StyledSection>
)

export default Section
