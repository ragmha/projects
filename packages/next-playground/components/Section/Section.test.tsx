import Section from './Section'
import { render } from '@utils/test'

describe('Section', () => {
  it('should match the snapshot', () => {
    const { container } = render(<Section>Info</Section>)
    expect(container).toMatchSnapshot()
  })
})
