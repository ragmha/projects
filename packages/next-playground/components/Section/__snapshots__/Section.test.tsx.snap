// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section should match the snapshot 1`] = `
<div>
  <section
    class="sc-AxjAm fnqROF"
  >
    Info
  </section>
</div>
`;
